File: languages/java/hello-world.java
import willyhorizont.runtime.Any;
import willyhorizont.runtime.JsLikeArray;
import willyhorizont.runtime.Utils;

@SuppressWarnings("unchecked")
public class Main {
    public static void main(String[] args) {
        System.out.println("Hello, World!");

        Any something;
        something = new JsLikeArray(1, 2, 3);

        System.out.println(something);
        System.out.println(something.at(0));
        System.out.println(something.squareBracket(2));
        System.out.println(((JsLikeArray) something).length);
        System.out.println(Utils.getType(something));
    }
}

File: languages/zig/hello-world.zig
const std = @import("std");
const xl = @import("willyhorizont/runtime/runtime.zig");

pub fn main(init: std.process.Init) !void {
    const gpa = init.gpa;

    std.debug.print("hello, world\n", .{});
    xl.print(gpa, .{}, .{ xl.Type{ .String = "hello, world" } });
}
